WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -18,6 +18,9 @@
 
 namespace WalletWasabi.Daemon.Rpc;
 
+[SuppressMessage("ReSharper", "CoVariantArrayConversion")]
+[SuppressMessage("ReSharper", "MemberCanBePrivate.Global")]
+[SuppressMessage("ReSharper", "UnusedMember.Global")]
```

Comment:
Yes it is. All of these methods are **Unused** because they are invoked dynamically by the RPC server. This is exactly the same that happens with all the WebApiController's methods. For the same reason all of these methods could be private but I prefer to have them public as a way to make clear that they can be invoked by something while private methods cannot.

Finally,  we can generate types for each return type instead of returning `object[]` but the only good thing that would come from that is the suppression of the warnings about CoVarianArrayConversion. I preferred to  disable the warning instead of creating types that would never been used for anything.

Reply:
It should be hidden because they are wrong.